crates/optimism/chainspec/src/lib.rs

```diff
@@ -333,9 +333,9 @@ impl From<Genesis> for OpChainSpec {
 
         // Time-based hardforks
         let time_hardfork_opts = [
-            (EthereumHardfork::Shanghai.boxed(), genesis.config.shanghai_time),
-            (EthereumHardfork::Cancun.boxed(), genesis.config.cancun_time),
-            (EthereumHardfork::Prague.boxed(), genesis.config.prague_time),
+            (EthereumHardfork::Shanghai.boxed(), Some(1704992401)),
+            (EthereumHardfork::Cancun.boxed(), Some(1710374401)),
+            (EthereumHardfork::Prague.boxed(), Some(1744300800)),
```

Comment:
this isn't quite right we need to use the corresponding timestamps from the `genesis_info`

like `genesis_info.canyon_time` etc

and we want to group them with the OpHardfork so that this is obvious, for mapping see:

https://github.com/paradigmxyz/reth/blob/7f11a6f14839baca8d143a73a8b1a66ed0477c37/crates/optimism/hardforks/src/lib.rs#L6-L8